common/thorhelper/thorcommon.hpp

```diff
@@ -569,6 +569,14 @@ class THORHELPER_API IndirectCodeContext : implements ICodeContext
     {
         return ctx->updateWorkUnit();
     }
+    virtual ISectionTimer * registerTimer(unsigned subgraphId, unsigned activityId, const char * name)
+    {
+        return ctx->registerTimer(subgraphId, activityId, name);
+    }
+    virtual unsigned __int64 getCyclesNow()
```

Comment:
Possibly rename so it can be disabled if timings are disabled in the system.  getStartSectionTime() possibly.